### PR TITLE
Fixed null file path return (specific to music visualizer)

### PR DESCRIPTION
**Pull request by elisa-lj11**
Test plan:

1. Download music visualizer app: https://hifi-content.s3.amazonaws.com/elisalj/music_visualizer/musicVisualizer.js and test sound: https://hifi-content.s3.amazonaws.com/elisalj/music_visualizer/test_sounds/UpbeatFunk_mono.wav
2. Open the app and click "Choose Audio File"
3. Select "Upbeat Funk"
4. Click on any particle, you should hear the music and the particle should "dance"

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4740/HighFidelity-Beta-PR11767-1ceec174052aa7295aa170738c74cb8a3bea34e0.dmg

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4742/HighFidelity-Beta-PR11767-1ceec174052aa7295aa170738c74cb8a3bea34e0.dmg

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4747/HighFidelity-Beta-PR11767-1ceec174052aa7295aa170738c74cb8a3bea34e0.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4747/HighFidelity-Beta-PR11767-1ceec174052aa7295aa170738c74cb8a3bea34e0.dmg

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

Download music visualizer app and test sound provided. **Check**
Open the app and click "Choose Audio File" **Check**
Select "Upbeat Funk" **Check**
Click on any particle, you should hear the music and the particle should "dance" **Check**

All **Pass**, tested the sound with a few different particles :) Very funky!

File: libraries/ui/src/OffscreenUi.cpp [hunk 1]
```diff
@@ -720,7 +720,8 @@ QString OffscreenUi::fileDialog(const QVariantMap& properties) {
         return QString();
     }
     qCDebug(uiLogging) << result.toString();
-    return result.toUrl().toLocalFile();
+    // originally was result.toUrl().toLocalFile();, but toLocalFile() is not working
+    return result.toString();
```
Comment (sethalves) [inline libraries/ui/src/OffscreenUi.cpp]: Is this a situation where we specifically didn't want over-the-network urls to work?

**Pull request merged**